Automattic/newspack-plugin
# Onboarding: add Design step

### All Submissions:






### Changes proposed in this Pull Request:



Add the Design step of the new Onboarding. Also, the UI is reused in the Design Wizard - this can be removed though. 

Some features to be tackled separately added to https://github.com/Automattic/newspack-plugin/issues/852

### How to test the changes in this Pull Request:

1. Visit the onboarding, Design step (`/wp-admin/admin.php?page=newspack-setup-wizard#/design`)
2. In different tabs, open the Design Wizard and the Customizer
3. Edit the options, obeserve they can be set & read in all three UIs
4. Observe the "Preview Site" floating button saves the settings & opens the preview - but it's available only in the onboarding 

### Other information:
